packages/react/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { shorthands, mergeClasses, createDOMRenderer } from '@griffel/core';
+export { shorthands, mergeClasses, createDOMRenderer, GriffelRenderer } from '@griffel/core';
```

Comment:
As there is a separate `export type` on the next line, shouldn't this go there?

Reply:
Oops, good catch 😨